supportedBackends/htcondor/src/main/scala/cromwell/backend/impl/htcondor/HtCondorInitializationActor.scala

```diff
@@ -0,0 +1,65 @@
+package cromwell.backend.impl.htcondor
+
+import cromwell.backend.BackendLifecycleActor.WorkflowAbortResponse
+import cromwell.backend.validation.RuntimeAttributesKeys._
+import cromwell.backend.validation.Validation._
+import cromwell.backend.{BackendConfigurationDescriptor, BackendWorkflowDescriptor, BackendWorkflowInitializationActor}
+import wdl4s.util.TryUtil
+import wdl4s.{Call, ThrowableWithErrors, WdlExpression}
+
+import scala.concurrent.Future
+import scala.util.{Failure, Success}
+import scalaz.Scalaz._
+
+class HtCondorInitializationActor(override val workflowDescriptor: BackendWorkflowDescriptor,
```

Comment:
This is just an example... once I have a validated model, I will continue with other backends.